client/src/components/modals/InviteUser.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+import Modal from 'react-modal';
+
+require('../../styles/DashboardModal.scss');
+
+export default function InviteUser(props) {
+  const { invitedUserEmail, isOpen, onChange, onClose, onInviteUser } = props;
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      contentLabel="userInviteModal"
+      style={{
+        content: {
+          width: 500,
+          height: 300,
+          marginLeft: 'auto',
+          marginRight: 'auto',
+          borderRadius: 0,
+          border: '0.1rem solid rgb(223, 244, 234)',
+        },
+        overlay: {
+          zIndex: 1000,
+          backgroundColor: 'rgba(0,0,0,0.6)',
+        },
+      }}
+    >
+      <div className="DashboardModal">
+        <div className="InviteUserModal">
+          <h2 className="modalTitle">Invite User</h2>
+          <div
+            className="close clickable"
+            onClick={onClose}
+          >
+          +
+          </div>
+          <div className="contents">
+            <p>Please enter the email address you would like to invite below.</p>
+            <form onSubmit={onInviteUser}>
```

Comment:
@peeb This is problematic as it will result in a page refresh, which I don't think we want/need here.

There are a couple of ways resolve this.

1. Quick fix is to add `preventDefault` to `onInviteUser`
2. We don't tend to use `<form ..>` , instead just handle the form state manually and have an ordinary `<button ..>` to "submit". There are scss for input fields and buttons to keep the styling consistent which I can look up if you pick this approach.